packages/web-pkg/src/helpers/resource/icon.ts

```diff
@@ -1,4 +1,12 @@
-import { IconType } from './types'
+export type IconFillType = 'fill' | 'line' | 'none'
+export type IconType = {
+  name: string
+  color?: string
+  fillType?: IconFillType
+}
+
+export type OcResourceIconMapping = Record<'mimeType' | 'extension', Record<string, IconType>>
```

Comment:
Small nitpick: If we keep the provide/inject of icon mappings, these two lines should also not contain `oc`-prefixes anymore.